pkg/hac/hac.go

```diff
@@ -0,0 +1,85 @@
+package hac
+
+// NooBaa Component Rescheduling
+// see doc/high-availability-controller.md
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	"github.com/noobaa/noobaa-operator/v5/pkg/options"
+	"github.com/noobaa/noobaa-operator/v5/pkg/util"
+)
+
+// Name of this controller
+const Name = "high-availability-controller"
```

Comment:
why Name defined here? isn't it just used when creating the controller?

Reply:
Controller Name variable is referenced twice:
- in this file, line 43, in NewHAC() for logger configuration  
- in pkg/controller/ha/ha_controller.go, line 59, Add(), see https://github.com/baum/noobaa-operator/blob/37980cba41350d3c74e4f1589c558f6f9aab8a20/pkg/controller/ha/ha_controller.go#L59